src/main/java/ticTacToe/game/GameFlow.java

```diff
@@ -35,35 +34,24 @@ public void runGame() {
     }
 
     private void setUpPlayers() {
+        String opponentOptionNumber = ui.chooseOpponent();
```

Comment:
Here and throughout the PR you refer to a `String` as a `...Number`.
That is very interesting.
Strings _can_ contain numbers ("9") but also loads of other stuff ("foo").
If on the other hand you were using `int` for the opponentOption, numbers still work (9) but `"foo"` won't be allowed and be caught by the compiler.